presto-main/src/main/java/com/facebook/presto/server/SessionSupplier.java

```diff
@@ -15,8 +15,14 @@
 
 import com.facebook.presto.Session;
 import com.facebook.presto.spi.QueryId;
+import com.facebook.presto.spi.session.SessionPropertyConfigurationManagerFactory;
 
 public interface SessionSupplier
 {
     Session createSession(QueryId queryId, SessionContext context);
+
+    void addSessionPropertyConfigurationManager(SessionPropertyConfigurationManagerFactory sessionConfigFactory);
```

Comment:
Let's shorten to `addConfigurationManager`. This is the "session supplier" so saying "session property" seems redundant.